### DISCUSSION TITLE
Copilot and Copilot chat extensions keep disabling on Visual Studio 17.10.0+

### DISCUSSION THREAD

**Discussion by MPITech** [community/community#118458] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Both extensions keep disabling, stating that they are not compatible with Visual Studio Pro 17.10.0.

**Comment by ayim** [comment]
Hey, this sounds odd to us.
Please use the Report a Problem feature in Visual Studio 2022 feedback so that we can take a closer look.

![image](https://github.com/community/community/assets/10951869/803c6b46-dfd7-418a-ad75-8260f80bc2bb)


--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio

**Comment by MPITech** [comment]
I updated from VS 17.10.0 Preview 2 to Preview 3 just now and since it drops the need for the separate extensions, it is now working again.  Perhaps the release of Preview 3 has to do with it.

**Comment by SorceryAdmin** [comment]
I'm also experiencing same issue, but is in a loop every time that visual studio opens even on safe mode it says copilot need to be disabled and then restart, I can't work at all
